release-notes/opensearch-security.release-notes-2.4.0.0.md

```diff
@@ -0,0 +1,15 @@
+## 2022-11-04 Version 2.4.0.0
+
+Compatible with OpenSearch 2.4.0
+
+### Enhancements
+* Add install_demo_configuration Batch script for Windows ([#2161](https://github.com/opensearch-project/security/pull/2161))
+* Add CI for Windows and MacOS platforms ([#2190](https://github.com/opensearch-project/security/pull/2190))
+
+### Bug Fixes
+* Point in time API security changes ([#2094](https://github.com/opensearch-project/security/pull/2094))
+* Fix windows encoding issues ([#2206](https://github.com/opensearch-project/security/pull/2206))
+
+### Maintenance
+* Add groupId = org.opensearch.plugin ([#2158](https://github.com/opensearch-project/security/pull/2158))
+* Roles yml changes for security-analytics plugin ([#2192](https://github.com/opensearch-project/security/pull/2192)
```

Comment:
Missing the following:
- https://github.com/opensearch-project/security/pull/2177
- https://github.com/opensearch-project/security/pull/2099
- https://github.com/opensearch-project/security/pull/2123
- https://github.com/opensearch-project/security/pull/2093
- https://github.com/opensearch-project/security/pull/2091

Found by inspecting:
https://github.com/opensearch-project/security/commits/2.4

Reply:
They are not tagged as v2.4.0 so I did not think they were supposed to be included in the release notes?

Reply:
I see a few others to add as well. I've crossed out the one's already in the release notes here.

Below is the output of `git log 0aedd61120364d89ac32450ebcc80307f2da6ee5..HEAD --pretty=format:"* %s"` where 0aedd61120364d89ac32450ebcc80307f2da6ee5 is the SHA of the commit that is tagged as 2.3.0.0. You can also reference the tag 2.3.0.0 if you fetch the tags from upstream using `git fetch upstream --tags`. 

~~* roles yml changes for security-analytics plugin (#2192)~~
~~* Point in time security changes (#2094)~~
~~* Fix windows encoding issues (#2206)~~
~~* Add CI for Windows and MacOS platforms (#2190)~~
~~* [Backport 2.x] Add install_demo_configuration Batch script for Windows (#2203)~~
* Add bcpkix-jdk15on runtimeOnly dependency to read keys with bouncycastle (#2191)
*  [FasterXML/Woodstox] Woodstox Version Bump to 6.4.0 (#2197)
* Patch bump for scala dependency (#2187)
* Address CVE-2022-42889 by updating commons-text (#2177)
~~* add group = org.opensearch.plugin (#2158)~~
* Log deprecation message on legacy ldap pool settings (#2099)
* Upgrade Kafka Client to 3.0.2 (#2123)
* Allow custom return attributes (#2093)
* Make ldap pool period and idle time configurable (#2091)
~~* Increment version to 2.4.0.0 (#2084)~~ -- Does not need to be included in release notes
* Fix the inconsistent variable for DLS (#2056)

Reply:
Should be all set. #2056 is from the end of August it would have been 2.3 I believe.

Reply:
#2056 was backported after the 2.3 release. I don't see the commit on the 2.3 branch.

Reply:
Looks good otherwise. Thank you for putting this together @scrawfor99!